src/pages/index.astro

```diff
@@ -1,8 +1,52 @@
 ---
-import Homepage from '../components/Homepage.vue';
 import BaseLayout from '../layouts/BaseLayout.astro';
+import FeaturedGuideCard from '../components/FeaturedGuideCard.astro';
+import { getCollection } from 'astro:content';
+import Prose from '../components/Prose.vue';
+export async function getStaticPaths() {
+  const guides = await getCollection('guides');
+  return guides.map(entry => ({
+    params: { slug: entry.slug },
+    props: { entry },
+  }));
+}
+
+const guides = await getCollection('guides');
+const featuredGuides = guides.filter((guide) => guide.data.featured);
 ---
 
 <BaseLayout>
-  <Homepage />
+  <div class="desktop-gutters px-4 sm:px-8 my-32">
+    <Prose class="max-w-none">
+      <h1 class="type-display">
+        <b>Centrapay Docs</b>
+      </h1>
+      <p class="type-body-1 max-w-[720px]">
+        Welcome to Centrapay! We enable you to transact fiat currency,
+        giftcards, digital assets and vouchers via your point of sale,
+        payment terminal, shopping cart or unattended device. We
+        accomplish this via our APIs which allow merchants, customers,
+        terminals, and smart wallets to interact with each other.
+      </p>
+
+      <p class="type-body-2 max-w-[720px]">
+        For more help getting setup, contact us via email at
+        <a href="mailto:integrations@centrapay.com">
+          integrations@centrapay.com.
+        </a>
+      </p>
+      <h2 class="type-headline-2">Featured Docs</h2>
+    </Prose>
+      <div class="grid md:grid-cols-2 lg:grid-cols-3 gap-4 lg:gap-6 mt-8">
+        { featuredGuides.map((guide) => (
```

Comment:
In the future we will want to control the order of guides, we don't know what should control that yet so it should be fine to just have them ordered alphabetically.

Reply:
The farmlands guides should not be first